Makefile

```diff
@@ -53,8 +59,8 @@ deploy: manifests
 
 # Generate manifests e.g. CRD, RBAC etc.
 .PHONY: manifests
-manifests: $(CONTROLLER_GEN)
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+manifests: controller-gen
```

Comment:
Revert please and I'm guessing some of the other diffs in the PR might end up getting undone as well (different controller-gen version).